src/simulations/spatial_parameters.jl

```diff
@@ -8,6 +8,60 @@ function mean(x::AbstractArray{T}) where {T}
     sum(x) / length(x)
 end
 
+apply_threshold(field, value) =
+    field > value ? eltype(field)(1) : eltype(field)(0)
+
+"""
+    landsea_mask(
+        surface_space;
+        resolution = "60arcs",
+        threshold = 0.5,
+        regridder_type = :InterpolationsRegridder,
+        extrapolation_bc = (
+            Interpolations.Periodic(),
+            Interpolations.Flat(),
+            Interpolations.Flat(),
+        ),
+    )
+
+Reads in the default Clima 60arcsecond land/sea mask, regrids to the
+`surface_space`, and treats any point with a land fraction < threshold
+as ocean.
+
+A 1degree (resolution = "1deg") and 30arcsecond (resolution = "30arcs") mask
+are also available.
+"""
+function landsea_mask(
```

Comment:
I wonder if this should be in utils.jl instead, since it could apply to other fields in addition to spatial parameters

Reply:
ok! Ill move it there